elixir_buildpack.config

```
@@ -0,0 +1,3 @@
+elixir_version=1.10.1
```

```suggestion
elixir_version=1.10.2
```
The last one from february is `1.10.2`:
https://github.com/elixir-lang/elixir/releases/tag/v1.10.2

---

nice! I will update with a separated PR :+1: